src/sx/README.md

```diff
@@ -194,9 +196,12 @@ export default class MyDocument extends Document {
 This will add the following css to your stylesheet:
 
 ```css
+html,
 body {
+  font-size: 16px;
```

Comment:
It should be sufficient to have it only on the HTML tag

Reply:
Actually, I guess it should be 

```css
html {
  font-size: 16px;
}
body {
  font-size: 1rem;
}
```

Reply:
Actually, even setting `font-size: 16px` on the HTML prevents scaling font-sizes. It should be set to `100%` or not set at all.